src/components/implementation/no_interface/llbooter/Makefile

```diff
@@ -1,7 +1,7 @@
 COMPONENT=llbooter.o
 ASM_OBJS=s_stub.o
 COMPONENT=llboot_comp.o
-INTERFACES=
+INTERFACES=fault_handler
```

Comment:
Name choices: `faults`, `faultmgr` and of course `fault_handler` here. I actually like having the API name contain interface name as a prefix but again that's a personal preference, just wanted to mention that.